unittests/database_tests.cpp

```diff
@@ -87,4 +88,15 @@ BOOST_AUTO_TEST_SUITE(database_tests)
       } FC_LOG_AND_RETHROW()
    }
 
+   //test that find_secondary_key behaves like lowerbound
+   BOOST_AUTO_TEST_CASE_TEMPLATE( find_seconary_key, T, validating_testers ) {
```

Comment:
Seems like `api_tests.cpp` `db_tests` section would be a better place. Even better would to pull out all multi index db tests into a separate file and include this.